test/configCases/url/file-loader/webpack.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+	mode: "development",
+	output: {
+		urlModuleFilename: "images/file[ext]"
+	},
+	module: {
+		rules: [
+			{
+				test: /\.png$/,
+				loader: "file-loader",
+				options: {
+					name: "file-loader.[ext]"
```

Comment:
```suggestion
					name: "file-loader[ext]"
```

Even if it's a bit weird to not include parts of the filename or a hash in the name

Reply:
This test uses file-loader, so I think we should specify `file-loader.[ext]`.